client/src/sendQuery.ts

```diff
@@ -7,17 +7,21 @@ import {
   QueryBuilderV2,
   QueryV2,
 } from "@axiom-crypto/core";
-import { encodeFunctionData } from "viem";
+import { createPublicClient, encodeFunctionData, http } from "viem";
 import { getMaxFeePerGas } from "./axiom/utils";
-import { AxiomV2ClientOptions, AxiomV2SendQueryArgs } from "./types";
+import { AbiType, AxiomV2ClientOptions, AxiomV2SendQueryArgs } from "./types";
 import { encodeFullQueryV2 } from "@axiom-crypto/core/packages/tools";
+import { calculatePayment } from "./lib/paymentCalc";
+import { viemChain } from "./lib/viem";
+import { getAxiomV2Abi, getAxiomV2QueryAddress } from "./lib";
 
 export const buildSendQuery = async (input: {
   axiom: AxiomSdkCore;
   dataQuery: DataSubquery[];
   computeQuery: AxiomV2ComputeQuery;
   callback: AxiomV2Callback;
   caller: string;
+  provider: string;
```

Comment:
Is this function used anywhere else downstream?  This is a breaking change for axiom-std, so we should check if it is elsewhere

Reply:
was able to use the providerUri in `AxiomSdkCore` now, so no change needed in this input